subsys/bluetooth/controller/ll_sw/nordic/lll/lll.c

```diff
@@ -404,10 +404,13 @@ uint32_t lll_preempt_calc(struct evt_hdr *evt, uint8_t ticker_id,
 	uint32_t diff;
 
 	ticks_now = ticker_ticks_now_get();
-	diff = ticker_ticks_diff_get(ticks_now, ticks_at_event);
+	diff = ticks_now - ticks_at_event;
```

Comment:
why not keep the original function-call?

Reply:
The masking is not required as the MSbit is now checked. If MSBit (bit 23) is set then `ticks_now` is less than `ticks_at_event`.